distutils/command/install.py

```diff
@@ -68,8 +68,8 @@
     INSTALL_SCHEMES['nt_user'] = {
         'purelib': '{usersite}',
         'platlib': '{usersite}',
-        'headers': '{userbase}/{implementation}{py_version_nodot}/Include/{dist_name}',
-        'scripts': '{userbase}/{implementation}{py_version_nodot}/Scripts',
+        'headers': '{userbase}/{implementation}{py_version_nodot_plat}/Include/{dist_name}',
+        'scripts': '{userbase}/{implementation}{py_version_nodot_plat}/Scripts',
```

Comment:
Are these schemes already updated in `sysconfig` for Python 3.7+? If so, perhaps the best course of action would be to remove this scheme altogether and rely on the definition as found in `sysconfig`.

Reply:
Alright. Nevermind. I see now how these are needed, because py_version_nodot_plat was only added in Python 3.10.